### ISSUE TITLE
ValueError: Invalid format string

### ISSUE THREAD

**Issue by SkinAir** [agrawalsmart7/scodescanner#7]
logger.info("Starting the scanner - " + str(time.strftime("%Y-%m-%d-%hh-%MM-%ss")) +'\n')
ValueError: Invalid format string
scscanner.py line:39
I changed it to logger.info("Starting the scanner - " + str(time.strftime("%Y-%m-%d")) +'\n')
But I'm not sure if it's accurate,Because there is no error in the operation, but there is no result
![image](https://user-images.githubusercontent.com/71880910/194897945-316f7602-ea3d-4eeb-bd96-25ff21e7f105.png)

**Comment by SkinAir** [comment]
![image](https://user-images.githubusercontent.com/71880910/194898356-ac00bb25-f449-4f2c-9311-1159d75415e6.png)
There seems to be some new mistakes

**Comment by agrawalsmart7** [comment]
Could you be able to run the tool on a basic folder - For ex - Try this repo. https://github.com/oktadev/sql-injection-in-php

**Comment by SkinAir** [comment]
> 您能否在基本文件夹上运行该工具？- 对于前 - 试试这个 repo。https://github.com/oktadev/sql-injection-in-php
> 
> 也请给出 -o test.txt 文件名
> 
> 另请检查您是否使用 Python3 - 使用 Python 3.9.10

![image](https://user-images.githubusercontent.com/71880910/194905036-7842bef3-700c-4db5-99fa-0a914f562ac3.png)
You mean, execute the following command
python scscanner.py php sql-injection-in-php -o test.txt

**Comment by agrawalsmart7** [comment]
No, try this - 

 python3 scscanner.py php --folder sql-injection-in-php -o test.txt

**Comment by SkinAir** [comment]
![image](https://user-images.githubusercontent.com/71880910/195024419-914d4e91-b5f9-404c-a975-7a25cfb25a64.png)
Sorry, I can't find the cause of the problem

**Comment by agrawalsmart7** [comment]
Yes, I guess the problem resides in %hh, Windows supports %tt. Not sure, but I will check this issue and update the code accordingly.
